src/components/accounts/SetRecoveryInfoFormEmail.js

```diff
@@ -0,0 +1,112 @@
+import React, { Fragment } from 'react'
```

Comment:
looks like this uses tabs for formatting?

it's a bit in a mess now (some files have 3 (!) spaces, some have 4 spaces). Let's stick to 4 spaces to be consistent with nearlib and near-shell

Reply:
Yes there's currently an inconsistency with spaces. Most files seem to have 3 spaces, but will stick to 4 spaces going forward.